index.php

```diff
@@ -5,7 +5,7 @@
 	<body>
 		<h1>
 			<?php 
-				if(isset($_GET['name']) && isset($_GET['lastname'])) { 
+				if(isset($_GET['name']) && !empty($_GET['name']) && isset($_GET['lastname']) && !empty($_GET['lastname'])) { 
```

Comment:
consider wrapping the checks for isset() and !empty in a function, maybe call it isPresent()